flexbridge

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Run command in environment
+
+set -e -o pipefail
+
+scriptdir="$(dirname "$0")"
+prefix=$(cd "$scriptdir/../.."; /bin/pwd)
+
+cd "$(dirname "$0")"
+. environ
+
+(
```

Comment:
I'm a bit confused by the use of a subshell here. Why use a subshell in parentheses here? Ah, is it so XDG_DATA_HOME and FB_SHARE  don't stay set outside of the subshell?